app/scripts/modules/core/src/managed/Environments.module.css

```diff
@@ -27,16 +27,27 @@
 }
 
 .mainContent {
-  display: flex;
-  margin-top: 24px;
+  display: grid;
+  grid-template-columns: minmax(300px, 3fr) 8fr;
```

Comment:
if you're not familiar with grid, tld;r the sidebar has a minimum width of 300px, but if given the space will attempt to take 3 "fractional units" of the overall width. The detail column will always try to take 8 fractional units (so at larger widths the sidebar is 3/8ths of the detail column) but will go smaller once the sidebar hits the minimum width.

Reply:
Thanks for the explanation!